api/admin_types.go

```diff
@@ -18,6 +18,8 @@ type HealthStatus struct {
 	BestBlockTime        *time.Time `json:"bestBlockTime"`
 	PeerCount            int        `json:"peerCount"`
 	IsNetworkProgressing bool       `json:"isNetworkProgressing"`
+	Master               *string    `json:"nodeMaster"`
```

Comment:
I think it might make sense, given the other occurrences

```suggestion
	NodeMaster               *string    `json:"nodeMaster"`
```